geth/params/defaults.go

```diff
@@ -12,6 +12,9 @@ const (
 	// StatusDatabase path relative to DataDir.
 	StatusDatabase = "status-db"
 
+	// DeduplicatorDatabase path relative to DataDir.
+	DeduplicatorDatabase = "status-deduplicator-db"
```

Comment:
why add new database?

Reply:
Sure, that is a good idea.
I'll take a look at reusing the database.